oe_showcase.post_update.php

```diff
@@ -1059,3 +1059,55 @@ function oe_showcase_post_update_00046(&$sandbox): void {
   }
   $role->save();
 }
+
+/**
+ * Delete the "Manage users" role and uninstall the "RoleAssign" module.
+ */
+function oe_showcase_post_update_00047() {
+  $role = Role::load('manage_users');
+  if ($role) {
+    $role->delete();
+  }
```

Comment:
We could change this for: 
```suggestion
    $role?->delete();
```